### PR TITLE
Solution for silent install on windows?

### LINKED ISSUE #3832: Package OpenStudio 2.5 With Qt IFW v3 For Silent Windows Installation
Hi Team,

Can someone please package OpenStudio 2.5 with QTIFW 3 to enable a remote silent deployment?
Or could someone supply an MSI installer for OpenStudio 2.5 with silent install switches?

The goal is to enable a silent Windows deployment.

Best,
mhickey3

### PR DESCRIPTION
**Pull request by SyZn**
Pull request overview
---------------------

 - Fixes #3832
 - This should fix the silent install issues on windows

Please read [OpenStudio Pull Requests](https://github.com/NREL/OpenStudio/wiki/OpenStudio-Pull-Requests) to better understand the OpenStudio Pull Request protocol.

### CODE REVIEW

File: install.qs [hunk 1]
```diff
@@ -1,6 +1,10 @@
 function Controller () {
   // silent install is not an option until QtIFW v3.0.1
   // gui.setSilent(true);
+  // On Windows uncomment the following 3 lines to make silent installation pass the install screen
+  //installer.installationFinished.connect(function() {
+  //  gui.clickButton(buttons.NextButton);
+  //});
```
Comment (jmarrec) [inline install.qs:7]:
```suggestion
  installer.autoRejectMessageBoxes();

  // silent install is not an option until QtIFW v3.0.1
  gui.setSilent(true);

  installer.setMessageBoxAutomaticAnswer("OverwriteTargetDirectory",
                                         QMessageBox.Yes);

  installer.installationFinished.connect(function() {
    gui.clickButton(buttons.NextButton);
  });
  // Uninstaller
  installer.uninstallationFinished.connect(function() {
    gui.clickButton(buttons.NextButton);
  });
  
```

**Comment by SyZn** [comment]
@jmarrec for me this throws an error on windows (I am installing the latest stable version 3.0.1).
The error is: "TypeError: Property 'setSilent' of object QInstaller::GuiProxy(0x496af00) is not a function"

**Comment by jmarrec** [comment]
> @jmarrec for me this throws an error on windows (I am installing the latest stable version 3.0.1).
> The error is: "TypeError: Property 'setSilent' of object QInstaller::GuiProxy(0x496af00) is not a function"

Thanks @SyZn, QtIFW is outdated on the Windows CI box. We'll fix this shortly.

**Comment by tijcolem** [comment]
Thanks for the contributuin @SyZn!  Some others have asked for this as well. I'll get these CI boxes updated here soon and then report back. You'll be able to test an installer before we merge this in PR in.  And thanks @jmarrec for sharing those scripts. I'll take a look.

**Comment by lgentile** [comment]
Hi @SyZn thank you for contributing to OpenStudio! In order for us to accept your contribution, please email me (luigi.gentile.polese@nrel.gov) with the IP Consent text available at https://www.openstudio.net/openstudio-contribution-policy. Thanks again!

**Comment by tijcolem** [comment]
QtIFW 3.2.2 is now on Mac and Windows build boxes. I replayed Windows CI and hitting a build error. Looking into that now. But once this passed CI, we can test out a package to verify the silent install works as expected.

**Comment by tijcolem** [comment]
@SyZn  Can you please test this installer? This has your changes.  
https://openstudio-ci-builds.s3-us-west-2.amazonaws.com/patch-1/OpenStudio-3.0.1-beta%2B7d4caf9401-Windows.exe

I tested using this setup script - https://raw.githubusercontent.com/NREL/OpenStudio/2.9.X-LTS/install.qs 

Need to uncomment the 3rd line `gui.setSilent(true);`

`./OpenStudio-3.0.1-beta+7d4caf9401-Windows.exe --verbose --script install.qs`
Works! 

Here's the version of IFW we are now using. 
```
./OpenStudio-3.0.1-beta+7d4caf9401-Windows.exe --version
IFW Version: 3.2.2, built with Qt 5.12.7.
Build date: Mar 10 2020
Installer Framework SHA1: c0732c40
```

Once you confirm we can merge this in.  Thanks for your contribution!

**Comment by jmarrec** [comment]
@tijcolem I updated the install.qs on this PR to use gui.setSilent already instead of just adding comments. https://github.com/NREL/OpenStudio/pull/4029/commits/c52bebca142e66f1499240d6fb70e0c2839b7ba9

This should work as is with https://raw.githubusercontent.com/NREL/OpenStudio/c52bebca142e66f1499240d6fb70e0c2839b7ba9/install.qs

**Comment by SyZn** [comment]
@tijcolem I've tested the installer (with @jmarrec's install.qs). It works!

**Pull request merged**